### PR TITLE
Apply Damian Edwards suggestions /7

### LINKED ISSUE #28888: Apply AOT suggestions
[these suggestions](https://github.com/dotnet/AspNetCore.Docs/pull/28826#pullrequestreview-1371845739)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: da8f22ab-0112-9e6b-77c9-42d72d13b9d8
* Version Independent ID: da8f22ab-0112-9e6b-77c9-42d72d13b9d8
* Content: [ASP.NET Core support for .NET native AOT](https://learn.microsoft.com/en-us/aspnet/core/native-aot/?view=aspnetcore-8.0&branch=pr-en-us-28826)
* Content Source: [aspnetcore/native-aot/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/native-aot/index.md)
* Product: **aspnet-core**
* GitHub Login: @mitchdenny
* Microsoft Alias: **midenn**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #28888 applies @DamianEdwards post merge suggestions

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/native-aot/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/afb3cda45cc13abe963dba3f3c1e6bf13722ee57/aspnetcore/native-aot/index.md) | [aspnetcore/native-aot/index](https://review.learn.microsoft.com/en-us/aspnet/core/native-aot/index?branch=pr-en-us-28889) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@captainsafia @eerhardt @JamesNK  please suggest a better TOC location.

**Comment by Rick-Anderson** [comment]
> > @mitchdenny where do you want this in the TOC?
> 
> @Rick-Anderson, @JamesNK mentioned in a side chat he recommends that we put it under > fundamentals:
> 
> https://learn.microsoft.com/en-us/aspnet/core/fundamentals/?view=aspnetcore-7.0&> tabs=windows
> 
> TOC name: Native AOT or Native AOT deployment
> 
> Sounds reasonable to me.

I think it's better under deployment.

**Comment by JamesNK** [comment]
> I think it's better under deployment.

While native AOT compilation happens at deployment, thinking about native AOT compatibility has to happen throughout the development of the app.

**Review by mitchdenny — APPROVED** [review]

File: aspnetcore/toc.yml [hunk 1]
```diff
@@ -216,6 +216,8 @@ items:
         uid: fundamentals/startup
       - name: Dependency injection (services)
         uid: fundamentals/dependency-injection
+      - name: Native AOT
+        uid: native-aot/index
```
Comment (JamesNK) [inline aspnetcore/toc.yml:220]: Should the page URL be updated? `fundamentals/native-aot`?

**Review by JamesNK — APPROVED** [review]

File: aspnetcore/fundamentals/aot.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ monikerRange: '>= aspnetcore-8.0'
 ms.author: midenn
 ms.custom: mvc
 ms.date: 04/10/2023
-uid: native-aot/index
+uid: fundamentals/aot
```
Comment (JamesNK) [inline aspnetcore/fundamentals/aot.md:9]: Other docs in this area include native-aot in the URL - https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/
```suggestion
uid: fundamentals/native-aot
```

**Pull request merged**